Dockerfile.konflux

```diff
@@ -18,23 +18,19 @@ ARG CI_CONTAINER_VERSION="unknown"
 
 FROM registry.redhat.io/ubi8/ubi-minimal@sha256:7583ca0ea52001562bd81a961da3f75222209e6192e4e413ee226cff97dbd48c AS stage
 
-# Define a build argument for the PNC list of built files
-ARG PNC_FILES_JSON
-RUN echo "Files to download: $PNC_FILES_JSON"
+RUN ls -la .
+RUN ls -la ..
+RUN ls -la ./cachi2/output
+RUN ls -la ./cachi2/output/deps
+RUN ls -la ./cachi2/cachi2.env
 
 # Install packages for the install script and extract archives
 RUN microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install -y unzip jq wget
```

Comment:
The above are most likely debug `ls` statements and probably can be removed. 

jq and wget can be removed as they are no longer needed.